lib/kitchen/driver/vagrant.rb

```diff
@@ -326,6 +328,9 @@ def load_needed_dependencies!
       # @raise [ActionFailed] if the Vagrantfile template was not found
       # @api private
       def render_template
+        config[:box_download_ca_cert] = File.expand_path(
+          config[:box_download_ca_cert], config[:kitchen_root]) unless
+            config[:box_download_ca_cert].nil?
```

Comment:
I'd move this to `finalize_config` which I believe will mean you can more easily test this (see comment below)